front/client/page/categoryItem.test.js

```diff
@@ -0,0 +1,43 @@
+import { Selector } from 'testcafe'
+import VueSelector from 'testcafe-vue-selectors'
+
+fixture `ItemAddDelete`
+  .page `http://localhost:5000/haskell`;
+
+test('Add New Item to category', async addItem => {
+  const addItemBtn = Selector('.add-item-btn')
+  const addItemInput = Selector('input[aria-label="Item name"]')
+  const itemSubmitBtn = Selector('.add-cat-submit')
+  
+  await addItem
+    .click(addItemBtn)
+    .typeText(addItemInput, 'testcafe-item')
+    // .pressKey('enter') - when you click on enter some strange page reload occur
+    .click(itemSubmitBtn)
+  for (let i = 0; Selector('.article-hd-textlg').length; i++) {
```

Comment:
Better to use `count` property of Selector (thats what they advice in TestCafe documentation)